spec/jobs/data_clear_jobs/clear_user_data_spec.rb

```diff
@@ -7,6 +7,8 @@
   let!(:conversation) { create(:live_chat_conversation, intervention: intervention) }
   let!(:user_intervention1) { create(:user_intervention, intervention: intervention, user: create(:user, :participant, :confirmed)) }
   let!(:user_intervention2) { create(:user_intervention, intervention: intervention, user: create(:user, :guest, :confirmed)) }
+  let!(:user_intervention3) { create(:user_intervention, intervention: intervention, user: create(:user, :confirmed, roles: ['predefined_participant'])) }
```

Comment:
Why did you define that the owner of this intervention is a predefined participant?